.golangci.yml

```diff
@@ -0,0 +1,31 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 10m
+
+  skip-dirs:
```

Comment:
This was copy-past from SPIRE? These directories do not exist in this repo. We shouldn't add exemptions until we need to.

Reply:
Yep, removed those directories